### ISSUE TITLE
latest (1.4.3) code - issues with column width  - help wanted

### ISSUE THREAD

**Issue by ssjogus** [AllenFang/react-bootstrap-table#237]
i have updated the plugin code to 1.4.3, and facing one issue with the width of the columns..

i have a table with selectRow='true' option.. 

Strangely even if i don't specify any width to any columns, the radio button column is getting a long width. I tried to give some width to second column, but it is not as per expectation..

any clue why this is happening?

here is the code and screenshot:

```
<BootstrapTable data={tableDataSet} pagination={true} condensed={true} search={true}  searchPlaceholder='Search...' selectRow={selectRowProp} height='196'>
 <TableHeaderColumn dataField="Id" isKey={true} hidden={true}> Item Id </TableHeaderColumn>
 <TableHeaderColumn dataField="Name" > Item Name </TableHeaderColumn>
 </BootstrapTable>
```

![table-with-radiobutton](https://cloud.githubusercontent.com/assets/8896732/12604234/b6dd3af0-c4df-11e5-8ea5-fd410cd20296.png)

**Comment by AllenFang** [comment]
@ssjogus, ok I'll check this out. Thanks.
btw, which origin version did you used? and is it happened on chrome?

**Comment by ssjogus** [comment]
i was using version 0.9.7.. then updated to 1.4.3. Its happening on chrome

ok, i will try with 1.4.5

**Comment by AllenFang** [comment]
@ssjogus, did you replace the css file of ```react-bootstrap-table```?

**Comment by ssjogus** [comment]
yes i did

**Comment by AllenFang** [comment]
ok, hope not to cache the css file in your browser, if the problem still remain when you upgrade to ```v1.4.5```, I'll check this out based on your example. Thanks

**Comment by AllenFang** [comment]
@ssjogus, I've tested it on chrome and safari, it's work.
![Imgur](http://i.imgur.com/TmkpO7F.png)

any issue tested on v1.4.5?

**Comment by AllenFang** [comment]
if problem still remain, please follow #227.  I'll redesign bootstrap table for fixing column width and some issue for styling. But need some time. so close this issue and follow up #227

**Comment by AllenFang** [comment]
HI @ssjogus, ```v2.0.0``` has been released, you can check #331 for more detail explanation, if problem still remain, please tell me on #331 or open an issue
